### PR TITLE
Added more quick support topics

### PR DESCRIPTION
**Pull request by DJHARDY2K21**
Added two new quick support topics:
- Too many HTTP Requests,
- How to download products.

### CODE REVIEW

File: src/bot/commands/quick_support.js [hunk 1]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
...
+        ],
+        support_contents: 'To fix max HTTP Requests, Remove some lighting that you might think is unnecessary!'
+    }, {
+        title: 'How do I download the products?',
+        searchable_queries: [
+            'download',
+            'download products'
+        ],
+        support_contents: 'To download your files, please visit the [website](https://inertia.lighting/products)!'
     }
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:146]:
```suggestion
    },
```

File: src/bot/commands/quick_support.js [hunk 2]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
...
+            'http limit'
+        ],
+        support_contents: 'To fix max HTTP Requests, Remove some lighting that you might think is unnecessary!'
+    }, {
+        title: 'How do I download the products?',
+        searchable_queries: [
+            'download',
+            'download products'
+        ],
+        support_contents: 'To download your files, please visit the [website](https://inertia.lighting/products)!'
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:145]:
```suggestion
        support_contents: 'To download your products, go to the [products page](https://inertia.lighting/products) on our website.\nMake sure to login with the correct discord account!',
```

File: src/bot/commands/quick_support.js [hunk 3]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
...
+        searchable_queries: [
+            'max http',
+            'http limit'
+        ],
+        support_contents: 'To fix max HTTP Requests, Remove some lighting that you might think is unnecessary!'
+    }, {
+        title: 'How do I download the products?',
+        searchable_queries: [
+            'download',
+            'download products'
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:143]:
```suggestion
            'downloading products',
```

File: src/bot/commands/quick_support.js [hunk 4]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
...
+        title: 'Max HTTP Requests',
+        searchable_queries: [
+            'max http',
+            'http limit'
+        ],
+        support_contents: 'To fix max HTTP Requests, Remove some lighting that you might think is unnecessary!'
+    }, {
+        title: 'How do I download the products?',
+        searchable_queries: [
+            'download',
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:142]:
```suggestion
```

File: src/bot/commands/quick_support.js [hunk 5]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
...
         support_contents: 'To enable HTTP Requests, open the \"Game Settings\" menu in Roblox Studio, click on the \"Security\" tab, enable \"HTTP requests\", then save changes.',
+    }, {
+        title: 'Max HTTP Requests',
+        searchable_queries: [
+            'max http',
+            'http limit'
+        ],
+        support_contents: 'To fix max HTTP Requests, Remove some lighting that you might think is unnecessary!'
+    }, {
+        title: 'How do I download the products?',
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:140]:
```suggestion
        title: 'How do I download products that I have purchased?',
```

File: src/bot/commands/quick_support.js [hunk 6]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
             'http requests',
         ],
         support_contents: 'To enable HTTP Requests, open the \"Game Settings\" menu in Roblox Studio, click on the \"Security\" tab, enable \"HTTP requests\", then save changes.',
+    }, {
+        title: 'Max HTTP Requests',
+        searchable_queries: [
+            'max http',
+            'http limit'
+        ],
+        support_contents: 'To fix max HTTP Requests, Remove some lighting that you might think is unnecessary!'
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:138]:
```suggestion
        support_contents: 'The current limitation for sending and receiving HTTP requests is 500 requests per minute.\nTo work around this, you need to find ths script(s) that are making too many requests, then remove those script(s).',
```

File: src/bot/commands/quick_support.js [hunk 7]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
             'http requests',
         ],
         support_contents: 'To enable HTTP Requests, open the \"Game Settings\" menu in Roblox Studio, click on the \"Security\" tab, enable \"HTTP requests\", then save changes.',
+    }, {
+        title: 'Max HTTP Requests',
+        searchable_queries: [
+            'max http',
+            'http limit'
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:136]:
```suggestion
            'number of requests exceeded limit',
```

File: src/bot/commands/quick_support.js [hunk 8]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
             'http requests',
         ],
         support_contents: 'To enable HTTP Requests, open the \"Game Settings\" menu in Roblox Studio, click on the \"Security\" tab, enable \"HTTP requests\", then save changes.',
+    }, {
+        title: 'Max HTTP Requests',
+        searchable_queries: [
+            'max http',
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:135]:
```suggestion
            'too many http requests',
```

File: src/bot/commands/quick_support.js [hunk 9]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
             'http requests',
         ],
         support_contents: 'To enable HTTP Requests, open the \"Game Settings\" menu in Roblox Studio, click on the \"Security\" tab, enable \"HTTP requests\", then save changes.',
+    }, {
+        title: 'Max HTTP Requests',
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:133]:
```suggestion
        title: 'How do I fix \"number of requests exceeded limit\"?',
```

File: src/bot/commands/quick_support.js [hunk 10]
```diff
@@ -129,6 +129,20 @@ const qs_topics = [
             'http requests',
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:129]:
```suggestion
            'enabling http requests',
```

**Review by MidSpike — CHANGES_REQUESTED** [review]
Please merge the suggestions that I requested

**Review by MidSpike — APPROVED** [review]

**Review by rossloika — APPROVED** [review]

**Review by WillFJS — APPROVED** [review]

**Pull request merged**